runhouse/rns/defaults.py

```diff
@@ -288,3 +288,15 @@ def data_collection_enabled(self) -> bool:
             return False
 
         return True
+
+    @property
+    def observability_enabled(self):
```

Comment:
following up on note above, i think we want something like this at the cluster level too, since this represents a global configuration